_authors/daniel.strauch.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Daniel Strauch
+username: daniel-strauch
```

Comment:
Magst du den bitte noch in daniel.strauch ändern, damit er zum Autoren-Kürzel passt?